csunplugged/topics/content/en/unplugged-programming/programming-challenges/calculate-exponents/calculate-exponents.md

```diff
@@ -0,0 +1,29 @@
+# Calculate the exponents of numbers
+
+## Requirement:
+
+Write a program that asks the user to enter 2 numbers (an integer for the base and a positive integer for the power) and calculates the results of the first number to the power of the second number. You can assume that the power is 0 or more.
+
+Note: The exponent of a number says how many times to use that number in a multiplication. (e.g. 83 = 8 × 8 × 8 = 512 so the exponent 3 says to use the 8 three times in a multiplication.)
+
+## Testing examples:
+
+Your program should display the outputs shown in this table for the given inputs provided;
+
+<table>
+  <tr>
+    <th>Input</th>
+    <th>Output</th>
+  </tr>
+  <tr>
+    <td>3<br>5</td>
+    <td>3^5 = 243</td>
+  </tr>
+  <tr>
+    <td>-2<br>4</td>
+    <td>-2^4 = 16</td>
+  </tr>
+</table>
+
+
+Click on the green flag, enter the inputs provided in the “testing examples” to see the expected output of your program.
```

Comment:
This line should not be displayed in the description section, as it's Scratch dependent.